website/docs/guides/alpha-manifest-migration-guide.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: "kubernetes"
+page_title: "Migrating `kubernetes_manifest` resources from the kubernetes-alpha provider"
+description: |-
+  This guide covers adopting `kubernetes_manifest` resources created using the kubernetes-alpha provider.
+---
+
+# The kubernetes_manifest resource
+
+Earlier this year we announced a new provider capable of managing any kind of Kubernetes resource, but more specifically Custom Resources via a manifest configuration that could be translated directly from Kubernetes YAML. This was released as the experimental [kubernetes-alpha](https://github.com/hashicorp/terraform-provider-kubernetes-alpha) provider.
+
+The `kubernetes_manifest` resource in now available in the official provider for Kubernetes. This guide walks through the actions needed to adopt existing `kubernetes_manifest` resources into configurations that use the Kubernetes provider.
+
+Depending on use-case, the following actions are needed to perform the migration.
+
+# Use Case A
+
+## Using only `kubernetes_manifest` resources
+
+If you've already been using the experimental `kubernetes-alpha` provider, you are likely falling in this use-case. If you have other `kubernetes_*` resources in your configuration, please also see [Use-case B](#use-case-b)
+
+In case your configuration only uses `kubrenetes_manifest`, but no other resources of the Kubernetes provider (e.g. `kubernetes_*` resources) follow these steps to migrate.
+
+**Make sure you don't have any existing state generated with the `kubernetes-alpha` provider!**
```

Comment:
I'm worried this requires a depth of knowledge many users won't have. Can you also describe how to check that they isn't any existing state generated with the `kubernetes-alpha` provider?

Reply:
Good point! Let me add that.